pkg/policies/binary/binary.go

```diff
@@ -41,6 +42,11 @@ type OrgConfig struct {
 
 	// Action defines which action to take, default log, other: issue...
 	Action string `yaml:"action"`
+
+	// IgnoreFiles is a list of file names to ignore. Any Binary Artifacts found
+	// with these names are allowed, and the policy may still pass. These are
+	// just the file name, not a full path. Globs are not allowed.
+	IgnoreFiles []string `yaml:"ignoreFiles"`
```

Comment:
any plans to add list of acceptable hashes? Hard to maintain...

Reply:
Probably not in allstar. Maybe with raw results or with scorecard validation.